packages/climate-ref-core/src/climate_ref_core/constraints.py

```diff
@@ -152,6 +152,12 @@ class RequireFacets:
     required_facets: tuple[str, ...]
     operator: Literal["all", "any"] = "all"
 
+    group_by: tuple[str, ...] = ()
```

Comment:
Perhaps the default should be None instead of an empty tuple to indicate that no grouping is happening? That would be similar to `DataRequirement.group_by`.